tests/integration-tests/tests/common/utils.py

```diff
@@ -153,16 +155,16 @@ def get_sts_endpoint(region):
     return "https://sts.{0}.{1}".format(region, "amazonaws.com.cn" if region.startswith("cn-") else "amazonaws.com")
 
 
-def run_system_analyzer(cluster, scheduler_commands_factory, request, partition=None):
+def run_system_analyzer(cluster, scheduler, request, partition=None):
     """Run script to collect system information on head and a compute node of a cluster."""
     out_dir = request.config.getoption("output_dir")
     local_result_dir = f"{out_dir}/system_analyzer"
-    compute_node_shared_dir = "/opt/parallelcluster/shared"
+    compute_node_shared_dir = "/shared"
```

Comment:
Let's add this assumption in the readme, saying that the run_system_analyzer feature assumes there is a /shared folder

Reply:
Added